lib/error.ex

```diff
@@ -0,0 +1,5 @@
+defmodule DateTimeParser.Error do
```

Comment:
Is it overkill to make this `ParseError` to be more specific?

Reply:
I couldn't think of anything better on the spot :) but yea if you have a suggestion I'll be happy to put it in.

Reply:
I made #26 so you can see if it looks like a good rename